json_annotation/lib/src/json_enum.dart

```diff
@@ -34,4 +35,16 @@ class JsonEnum {
   /// Note: the value for [JsonValue.value] takes precedence over this option
   /// for entries annotated with [JsonValue].
   final FieldRename fieldRename;
+
+  /// if `true`, enum comparison will be done using case-insensitive.
+  ///
+  /// The default, `false`, means enum comparison will be done using
+  /// case-sensitive.
+  final bool caseInsensitive;
+
+  factory JsonEnum.fromJson(Map<String, dynamic> json) {
+    final caseInsensitive = json['case_insensitive'] as bool?;
+    return // caseInsensitive == null ? null :
```

Comment:
Drop the comment. It's pretty clear.
```suggestion
    return
```

Reply:
Entire method removed anyway